schemas/panels/time-series/time-series.cue

```diff
@@ -22,11 +22,12 @@ import (
 }
 
 #visual: {
-	line_width?:   number & >=0.5 & <=10
-	point_radius?: number & >=0 & <=20
+	line_width?:   number & >=0.5 & <=4
+	point_radius?: number & >=0 & <=5
```

Comment:
Out of curiosity, any reasons for these specific numbers? I wonder if `point_radius` should be a little bigger, so you can have a more distinct point if you have the largest `line_width`.

Reply:
I got some feedback from @foxbat07 about reducing them since I had the max set way too high originally, but I agree when playing around with it, you almost can't even see the point when line_width is larger. I just increased the point_radius max a little bit to account for this.